tests/rules/no-identical-functions.test.ts

```diff
@@ -0,0 +1,219 @@
+import { RuleTester } from "eslint";
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2018 } });
+import rule = require("../../src/rules/no-identical-functions");
+
+ruleTester.run("no-identical-functions", rule, {
+  valid: [
+    {
+      // different body
+      code: `
+      function foo() {
+        console.log("Hello");
+        console.log("World");
+        return 42;
+      }
+
+      function bar() {
+        if (x) {
+          console.log("Hello");
+        }
+        return 42;
+      }`,
+    },
+    {
+      // few lines
+      code: `
+      function foo() {
+        console.log("Hello");
+        return 42;
+      }
+
+      function bar() {
+        console.log("Hello");
+        return 42;
+      }`,
+    },
+    {
+      // ignore not block statement body
```

Comment:
@stas-vilchik this case is mistyped? there are different braces

Reply:
that's the point